Sources/App/constants.swift

```diff
@@ -89,14 +90,18 @@ enum Constants {
     """
     
     static let feedback = """
+    You are at the end of the conversation. No more health measurements are being recorded at this point.
```

Comment:
We can reach this point when a user ends the conversation or re-calls the service on the same day. We should not this here (and that the model can probably figure this out by looking at the current voice context). I would indicate some instructions about this and maybe even inject a small system message in addition to this if the user calls the service again? We currently load the state somewhere and could inject this additional system message there that e.g. states the time when the user finished the last set of questions.

Reply:
This is needed, when the call is a repeated one, where there are no questions to begin with. Then the agent would just hallucinate questions without this. I haven't found another reliable way of doing this unfortunately, but this works pretty consistently in my testing.

Reply:
Sorry, might have not been clear here. This can be reached from two states (end of the conversation or repeat call). I was noting that we should add some additional context about the repeat call scenario here (could be a separate message) + some instructions about the use case here in the general prompt?

Reply:
Thank you! Had a moment there, but looks good otherwise.